pkg/apis/core/validation/validation.go

```diff
@@ -1039,6 +1039,15 @@ func validateProjectionSources(projection *core.ProjectedVolumeSource, projectio
 				}
 			}
 		}
+		if projPath := fldPath.Child("serviceAccountToken"); source.ServiceAccountToken != nil {
+			numSources++
+			if !utilfeature.DefaultFeatureGate.Enabled(features.TokenRequestProjection) {
+				allErrs = append(allErrs, field.Forbidden(projPath, "TokenRequestProjection feature is not enabled"))
+			}
+			if source.ServiceAccountToken.ExpirationSeconds < 10*60 {
```

Comment:
should limit maximum to integer values that round-trip reliably in json (e.g. 2^53 or less)

Reply:
I limited it to 2^32, which is about 2000 years.

Reply:
> In the year 4027, humanity is on the brink of extinction. The last remaining humans have been forced into hiding under the onslaught of _kubeflow_. Then one day the attacks stop, the machines go quiet. **overlord-service** didn't properly handle cache invalidation of its service account token nor did it implement smart health checks.

```
expirationSeconds: 4294967296  # This should never expire
```